extensions/positron-assistant/src/modelDefinitions.ts

```diff
@@ -19,7 +19,7 @@ export interface ModelDefinition {
  */
 export function getConfiguredModels(providerId: string): ModelDefinition[] {
 	const config = vscode.workspace.getConfiguration('positron.assistant');
-	const configuredModels = config.get<Record<string, ModelDefinition[]>>('configuredModels', {});
+	const configuredModels = config.get<Record<string, ModelDefinition[]>>('models.custom', {});
 	return configuredModels[providerId] || [];
 }
```

Comment:
Nit on naming: should we change configuredModels --> customModels in the various places, to match the setting rename?